### ISSUE TITLE
webpage will crash with addon complicated args

### ISSUE THREAD

**Issue by wonderflow** [oam-dev/kubevela#3308]
Hi @wangyikewxgm , I tried this new feature, but hit one issue.
1. input list arguments for 'vela addon enable'
```
vela addon enable velaux imagePullSecrets={myreg}
```
2.The list is stored in Secret
 `"imagePullSecrets": "[myreg]",`

3.curl api-server
`curl 127.0.0.1:8000/api/v1/addons/velaux/status`
this api can not parse imagePullSecrets to List. It is string in 
```
 "args": {
  "imagePullSecrets": "[myreg]",
  "repo": "hub.jdcloud.com",
  "serviceType": "NodePort",
  "version": "v1.2.3"
 },

```

4.if you open velaux addon's details page on VelaUX webpage
The webpage will crash.

_Originally posted by @StevenLeiZhang in https://github.com/oam-dev/kubevela/issues/3232#issuecomment-1046793020_

**Comment by StevenLeiZhang** [comment]
I have verified this issue, it is fixed by v1.2.4

**Comment by wonderflow** [comment]
@StevenLeiZhang Thanks Steven!
